Source/WebCore/bindings/js/JSCustomElementInterface.h

```diff
@@ -58,7 +58,7 @@ class JSCustomElementInterface : public RefCounted<JSCustomElementInterface>, pu
         return adoptRef(*new JSCustomElementInterface(name, callback, globalObject));
     }
 
-    Ref<Element> constructElementWithFallback(Document&, const AtomString&);
+    Ref<Element> constructElementWithFallback(Document&, const AtomString&, bool parserConstructElementWithEmptyStack = false);
```

Comment:
We should use a enum class instead of a boolean parameter.

Reply:
so `enum class ParserConstructElementWithEmptyStack { Yes, No }`?

Reply:
Ideally:
`enum class ParserConstructElementWithEmptyStack: bool { No, Yes };`

I'm not a huge fan of the name but I don't have a better proposal at the moment.